src/mod/python.mod/pycmds.c

```diff
@@ -65,6 +65,11 @@ static void cmd_python(struct userrec *u, int idx, char *par) {
   Py_ssize_t n;
   int i;
 
+  if (!(isowner(dcc[idx].nick)) && (must_be_owner)) {
```

Comment:
```suggestion
  if (!isowner(dcc[idx].nick) && must_be_owner) {
```

then it's fine

Reply:
can we automate this? let the CI check for superflous brackets?